crates/transport-http/Cargo.toml

```diff
@@ -51,6 +55,10 @@ hyper = [
     "dep:serde_json",
     "dep:tower",
     "dep:tracing",
+    "dep:alloy-rpc-types-engine",
+    "alloy-rpc-types-engine/jwt",
+    "alloy-rpc-types-engine/serde",
+    "dep:jsonwebtoken",
```

Comment:
this will now enable all of these by default, do we want this?

I think we can feature gate this with engine-jwt or similar